mail_room.gemspec

```diff
@@ -18,6 +18,7 @@ Gem::Specification.new do |gem|
   gem.require_paths = ["lib"]
 
   gem.add_dependency             "net-imap", ">= 0.2.1"
+  gem.add_dependency             "net-http", "~> 0.1"
```

Comment:
This needs Ruby 2.6.0. I suppose we can just demote this to a development dependency until Ruby 2.5 is no longer supported?

Reply:
I just went ahead and dropped Ruby 2.5 support.

Reply:
Hmm, `net-http` only is really needed to eliminate the warnings if `faraday` is used, so maybe it's better to keep it as a dev dependency.